stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/MessageAttachmentsContent.kt

```diff
@@ -34,16 +35,24 @@ import io.getstream.chat.android.compose.ui.util.hasLink
  * @param message The message that contains the attachments.
  * @param onLongItemClick Handler for long item taps on this content.
  * @param onImagePreviewResult Handler when the user selects a message option in the Image Preview screen.
+ * @param isQuote Determines if attachment is inside a quoted message or not.
  */
 @Composable
 public fun MessageAttachmentsContent(
     message: Message,
     onLongItemClick: (Message) -> Unit,
     onImagePreviewResult: (ImagePreviewResult?) -> Unit = {},
+    isQuote: Boolean = false,
```

Comment:
Maybe instead of breaking the behavior here, we can have a `QuotedMessageAttachmentsContent` composable that just focuses on showing quoted content?